docs/release-notes/8.0.asciidoc

```diff
@@ -1,6 +1,55 @@
 [[release-notes-header-8.0.0]]
 == 8.0
 
+[discrete]
+[[release-notes-8.0.0]]
+=== 8.0.0
+
+[discrete]
+[[post-upgrade-reqs-8.0.0]]
+==== Post upgrade requirements
+* Changes to the indicator match rule's default threat indicator path (<<rule-ui-advanced-params, *Indicator prefix override*>>) might require you to update existing rules, or create new ones, after upgrading to {stack} version 8.0.0. Be mindful of the following ({pull}118821[#118821]):
+** If an indicator match rule's default threat indicator path was not defined before upgrading, it will receive the default value `threatintel.indicator` during the upgrade process. This allows the rule to continue using indicator data ingested by {filebeat} version 7.x. If a custom value was defined before the upgrade, the value remains the same.
+** If an existing indicator match rule was configured to use threat indicator indices generated from {filebeat} version 7.x, updating the default threat indicator path to `threat.indicator` after you upgrade to {stack} version 8.0.0 and {agent} or {filebeat} version 8.0.0 configures the rule to use threat indicator indices generated by {agent} or {filebeat} version 8.0.0.
+** Separate rules must be created to query threat intelligence indices created by {filebeat} version 7.x and version 8.0.0 because different default threat indicator path values are required.
+
+[discrete]
+[[breaking-changes-8.0.0]]
+==== Breaking Changes
+* Removes the trusted application API. The trusted application interface maintains identical functionality but now uses the exception list API ({pull}120134[#120134]).
+* Removes the list endpoint metadata API ({pull}119401[#119401]).
+
+[discrete]
+[[new-features-8.0.0]]
+==== Features
+* Shows all historical alerts for a given rule in Rule Details, including those associated with previous versions of the rule ({pull}120053[#120053]).
+* Improves the Rules and Rule Monitoring tables and enables actions on the Rule Monitoring table ({pull}119644[#119644]).
+* The Threat Intelligence view supports {agent}, {filebeat}, and custom integrations ({pull}116175[#116175]).
+
+[discrete]
+[[bug-fixes-8.0.0]]
+==== Bug fixes and enhancements
+* Redesigns and improves the Rules and Exceptions tables ({pull}118940[#118940]).
+* Updates the mappings of the rule registry to ECS version 8.0.0, so that the detection rules can process ECS version 8.0.0 data ({pull}123012[#123012]).
+* Updates rule import functionality to include exceptions ({pull}118816[#118816]).
+* Improves how dates are displayed in the Rules and Exceptions tables ({pull}117643[#117643]).
+* Allows you to create runtime fields from, and add them to, the Alerts and Timeline tables ({pull}117627[#117627], {pull}114806[#114806]).
+* Enhances the *Data view* selection modal’s UI, and hides the *Data view* dropdown when no data is present ({pull}117601[#117601], {pull}119956[#119956]).
```

Comment:
Omit bold, since we're not directing the reader to interact with these UI elements.